public/styles/components/nav-primary/_nav-utility.scss

```diff
@@ -47,11 +47,23 @@
       }
     }
   }
+
+  .bsds-nav-logo-secondary {
+    width: auto;
+    max-height: space(4x);
+    margin: space(md) space(3x);
+  }
 }
 
 @media (min-width: $breakpoint-lg-min) {
   .bsds-nav-utility {
-    display: block;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+
+    &:has(> .bsds-nav-logo-secondary) {
```

Comment:
`:has()` is not yet supported in firefox. It will be in v121 which is live in the developer edition, but standard firefox is still on v120.

Reply:
nice catch!